### PR TITLE
Feature/appinsight

### PR DESCRIPTION
**Pull request by govTechSatish**

### CODE REVIEW

File: app/config/messaging.js [hunk 1]
```diff
@@ -1,5 +1,4 @@
 const sharedConfig = {
-  appInsights: require('applicationinsights'),
```
Comment (paulsimonandrews) [inline app/config/messaging.js]: We need this as it enables app insights inside the ffc-messaging module

**Review by paulsimonandrews — CHANGES_REQUESTED** [review]
Need appInsights in the message config.

File: app/config/messaging.js (hunk 1 above)
Comment (govTechSatish) [reply app/config/messaging.js]: Done.

**Review by paulsimonandrews — APPROVED** [review]
LGTM

**Pull request merged**